packages/expo-document-picker/src/types.ts

```diff
@@ -1,16 +1,50 @@
+// @needsAudit
+/**
+ *
+ */
 export type DocumentPickerOptions = {
+  /**
+   * The [MIME type](https://en.wikipedia.org/wiki/Media_type) of the documents that are available
+   * to be picked. Is also supports wildcards like `'image/*'` to choose any image. To allow any type
+   * of document you can use `'* /*'`.
```

Comment:
Does it ultimately end up rendered as `*/*`? 🤔 
I'm concerned about proposed notation 😓
I suggest using this notation instead:
```suggestion
   * of document you can use `'&#42;/*'`.
```

I see it working in this demo: https://www.typescriptlang.org/play?#code/PQKhCgAIUgFAnA9gB0QU3gGwJ6QCZoDGiBewaAdsaQFyQBkAxACwBMA3MFCF8RQM4AXSAENIAXkgBvAL7twQA

Reply:
* `'*/*'` - breaks TS and TypeDoc no matter how I have tried to escape this
* `'&#42;/*'` = I have tried to use this notation, also `&ast;` but markdown rendered do not unescape those entities and output matches the comment, so HTML entity conversion or unescape would be needed anyway

However I think we can switch the mechanism to replace the given HTML entity instead, which might be a little bit clearer for the docs author.